members/rest.py

```diff
@@ -0,0 +1,61 @@
+from rest_framework import viewsets
+from rest_framework.response import Response
+from rest_framework.permissions import IsAuthenticated
+from rest_framework import status
+from django.db.models import Q
+import django_filters
+from rest_framework import filters
+
+from members.serializers import *
+from members.filters import MemberFilter
+from members.permissions import MemberPermissions
+
+from core.models import User
+from core.utils import get_semester_of_date
+
+import datetime
+
+
+
```

Comment:
Reformatterer du koden av og til @matsasc? For ser ut som det er en del ekstra white space og slikt som ctrl+alt+l enkelt rydder opp, i hvert fall hos meg